### PR TITLE
Fix #711: Timeout for when initiating biometry

### LINKED ISSUE #711: Timeout for when initiating biometry
user: 2023041207, , activation ID: 9cd21a81-4878-4386-a9e4-376beed38d7c 

steps:

- Finish verification of documents user with identifier 202304120.
- On the screen Selfie verification press the button Continue

expected result: start of selfie scanning 
actual result: timeout error on fronted

Backend log:
```
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.366  INFO 1255870 --- [io-8080-exec-65] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : GetEciesDecryptorRequest received, application key: gfv6saZiwmtOAMk9iD1xyg==, activation ID: 9cd21a81-4878-4386-a9e4-376beed38d7c
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.370  INFO 1255870 --- [io-8080-exec-65] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : GetEciesDecryptorRequest succeeded
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.374  INFO 1255870 --- [io-8080-exec-68] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : VerifySignatureRequest received, activation ID: 9cd21a81-4878-4386-a9e4-376beed38d7c
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.377  INFO 1255870 --- [io-8080-exec-68] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : VerifySignatureRequest succeeded
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.385 DEBUG 1255870 --- [io-8080-exec-64] .a.o.i.s.IdentityVerificationRestService : Onboarding process will be locked using PESSIMISTIC_WRITE lock, 40904245-e6c0-42d0-8c25-728d21658872
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.385 DEBUG 1255870 --- [io-8080-exec-64] c.w.a.o.i.service.OnboardingServiceImpl  : Onboarding process will be locked using PESSIMISTIC_WRITE lock, process ID: 40904245-e6c0-42d0-8c25-728d21658872
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.389 DEBUG 1255870 --- [io-8080-exec-64] c.w.a.o.s.guard.ProcessIdentifierGuard   : Onboarding process will be locked using PESSIMISTIC_WRITE lock, 40904245-e6c0-42d0-8c25-728d21658872
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.393  INFO 1255870 --- [io-8080-exec-64] c.w.a.o.i.service.PresenceCheckService   : Selected DocumentVerificationEntity(id=f65e8a75-656a-40a6-8428-34ecc2ef9f23, type=ID_CARD, uploadId=e283ea9e91075b541e74e2ccd471c9d9) as the source of person photo, OwnerId(activationId=9cd21a81-4878-4386-a9e4-376beed38d7c, userId=mockuser_2023041207)
[2023-04-12 10:52:58] [info] 2023-04-12 10:52:58.456 DEBUG 1255870 --- [io-8080-exec-64] c.w.a.o.p.i.s.IProovRestApiService       : Calling /claim/enrol/token userId=GIV4WDFOFJDDRZAPLDN7ZAOGYPCBAB3IEVPCXJ3JHT3OEABBCIHA, OwnerId(activationId=9cd21a81-4878-4386-a9e4-376beed38d7c, userId=mockuser_2023041207)
[2023-04-12 10:52:59] [info] 2023-04-12 10:52:59.178 DEBUG 1255870 --- [   scheduling-1] c.w.c.a.b.database.DatabaseAuditWriter   : Audit log batch insert succeeded, audit record count: 1, audit param is disabled
[2023-04-12 10:53:17] [info] 2023-04-12 10:53:17.333  INFO 1255870 --- [MessageBroker-3] o.s.w.s.c.WebSocketMessageBrokerStats    : WebSocketSession[0 current WS(0)-HttpStream(0)-HttpPoll(0), 0 total, 0 closed abnormally (0 connect failure, 0 send limit, 0 transport error)], stompSubProtocol[processed CONNECT(0)-CONNECTED(0)-DISCONNECT(0)], stompBrokerRelay[null], inboundChannel[pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0], outboundChannel[pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0], sockJsScheduler[pool size = 4, active threads = 1, queued tasks = 2, completed tasks = 599859]
.....
[2023-04-12 11:08:37] [info] 2023-04-12 11:08:37.193  WARN 1255870 --- [or-http-epoll-1] r.netty.http.client.HttpClientConnect    : [555b8a64-6, L:/10.0.0.4:56120 - R:eu3.rp.secure.iproov.me/20.103.224.156:443
] The connection observed an error
[2023-04-12 11:08:37] [info] io.netty.channel.unix.Errors$NativeIoException: recvAddress(..) failed: Connection timed out
[2023-04-12 11:08:37] [info] 2023-04-12 11:08:37.196 ERROR 1255870 --- [io-8080-exec-64] c.w.a.o.statemachine.StateMachineConfig  : Not accepted event PRESENCE_CHECK_INIT
[2023-04-12 11:08:37] [info] 2023-04-12 11:08:37.196  WARN 1255870 --- [io-8080-exec-64] .a.o.i.s.IdentityVerificationRestService : Missing one of important values to generate response entity, response=null, s
tatus=null
[2023-04-12 11:08:37] [info] 2023-04-12 11:08:37.198 ERROR 1255870 --- [io-8080-exec-64] c.w.a.o.e.DefaultExceptionHandler        : Error occurred when processing the request.
[2023-04-12 11:08:37] [info] org.springframework.transaction.UnexpectedRollbackException: Transaction silently rolled back because it has been marked as rollback-only
[2023-04-12 11:08:37] [info] #011at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:752) ~[spring-tx-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763) ~[spring-aop-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:708) ~[spring-aop-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at com.wultra.app.onboardingserver.impl.service.IdentityVerificationRestService$$EnhancerBySpringCGLIB$$b660a083.initPresenceCheck(<generated>) ~[classes/:1.5.0-SNAPSHOT]
[2023-04-12 11:08:37] [info] #011at com.wultra.app.onboardingserver.controller.api.IdentityVerificationController.initPresenceCheck(IdentityVerificationController.java:237) ~[classes/:1.5.0-SNAPSHOT]
[2023-04-12 11:08:37] [info] #011at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
[2023-04-12 11:08:37] [info] #011at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
[2023-04-12 11:08:37] [info] #011at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
[2023-04-12 11:08:37] [info] #011at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
[2023-04-12 11:08:37] [info] #011at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205) ~[spring-web-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:150) ~[spring-web-5.3.24.jar:5.3.24]
[2023-04-12 11:08:37] [info] #011at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117) ~[spring-webmvc-5.3.24.jar:5.3.24
]
```

Frontend error:
```
[APP] ERROR: Error Domain=NSURLErrorDomain Code=-1001 "The request timed out." UserInfo={_kCFStreamErrorCodeKey=-2102, NSUnderlyingError=0x2809ec930 {Error Domain=kCFErrorDomainCFNetwork Code=-1001 "(null)" UserInfo={_kCFStreamErrorCodeKey=-2102, _kCFStreamErrorDomainKey=4}}, _NSURLErrorFailingURLSessionTaskErrorKey=LocalDataTask <D0B40437-D2DF-4607-87D5-C21D892D7C14>.<332>, _NSURLErrorRelatedURLSessionTaskErrorKey=(
    "LocalDataTask <D0B40437-D2DF-4607-87D5-C21D892D7C14>.<332>"
), NSLocalizedDescription=The request timed out., NSErrorFailingURLStringKey=https://powerauth-dev.westeurope.cloudapp.azure.com/enrollment-server-onboarding/api/identity/presence-check/init, NSErrorFailingURLKey=https://powerauth-dev.westeurope.cloudapp.azure.com/enrollment-server-onboarding/api/identity/presence-check/init, _kCFStreamErrorDomainKey=4}
```

Subsequently, pressing the button Continue

Backend log:
```
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.006 DEBUG 1255870 --- [   scheduling-1] c.w.a.o.task.StateMachineTask            : Changing machine states in batch
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.222  INFO 1255870 --- [io-8080-exec-65] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : GetEciesDecryptorRequest received, application key: gfv6saZiwmtOAMk9iD1xyg==, activation ID: 9cd21a81-4878-4386-a9e4-376beed38d7c
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.226  INFO 1255870 --- [io-8080-exec-65] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : GetEciesDecryptorRequest succeeded
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.231  INFO 1255870 --- [io-8080-exec-68] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : VerifySignatureRequest received, activation ID: 9cd21a81-4878-4386-a9e4-376beed38d7c
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.234  INFO 1255870 --- [io-8080-exec-68] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : VerifySignatureRequest succeeded
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.242 DEBUG 1255870 --- [io-8080-exec-61] .a.o.i.s.IdentityVerificationRestService : Onboarding process will be locked using PESSIMISTIC_WRITE lock, 40904245-e6c0-42d0-8c25-728d21658872
[2023-04-12 10:53:51] [info] 2023-04-12 10:53:51.242 DEBUG 1255870 --- [io-8080-exec-61] c.w.a.o.i.service.OnboardingServiceImpl  : Onboarding process will be locked using PESSIMISTIC_WRITE lock, process ID: 40904245-e6c0-42d0-8c25-728d21658872
```

Frontend error:
The same

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: enrollment-server-onboarding/src/main/resources/application.properties [hunk 1]
```diff
@@ -134,6 +134,8 @@ enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyH
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyPort=0
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyUsername=
 enrollment-server-onboarding.document-verification.zenid.restClientConfig.proxyPassword=
+enrollment-server-onboarding.document-verification.zenid.restClientConfig.connectionTimeout=10000
+enrollment-server-onboarding.document-verification.zenid.restClientConfig.responseTimeout=60000
```
Comment (banterCZ) [inline enrollment-server-onboarding/src/main/resources/application.properties:138]: 🤔 Just thinking about database transaction timeout, which is 30 seconds by default.

**Review by banterCZ — APPROVED** [review]
Looks good.

File: enrollment-server-onboarding/src/main/resources/application.properties (hunk 1 above)
Comment (romanstrobl) [reply enrollment-server-onboarding/src/main/resources/application.properties:138]: Where do you get the information about 30 seconds? I believe the transaction timeout is longer than this, based on what we observed when testing the timeouts.

Comment (banterCZ) [reply enrollment-server-onboarding/src/main/resources/application.properties:138]: Sorry, I have messed it up with JTA. It seems that the default timeout for user transactions in Spring is -1.

**Pull request merged**